co.electriccoin.swiftlint.yml

```diff
@@ -0,0 +1,110 @@
+# This SwiftLint file is based on this great guidelines. 
+# https://github.com/raywenderlich/swift-style-guide
+
+excluded:
+  - ${PWD}/Carthage
+  - ${PWD}/Pods
+  - ${PWD}/DerivedData
+  - ${PWD}/xctemplates
+
+disabled_rules:
+  - discarded_notification_center_observer
+  - notification_center_detachment
+  - orphaned_doc_comment
+  - todo
+  - unused_capture_list
```

Comment:
Some more rules to consider disabling by default
-nesting (allow for types to be nested, common pattern in Swift)
-multiple_closures_with_trailing_closure
- generic_type_name (allow for arbitrarily long generic type names)